### PR TITLE
task/MM5-5833 Added and updated labels

### PR DESCRIPTION
**Pull request by Svane20**

### CODE REVIEW

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl [hunk 1]
```diff
@@ -148,11 +148,27 @@ resource configservice_label duplicate_asset_modal_next {
...
+  ]
+}
+
+resource configservice_label duplicate_asset_modal_do_for_all {
+  key = 'DUPLICATE_ASSET_MODAL_DO_FOR_ALL'
+  group = 'Duplicate Assets'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Do for all'
```
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Repeat this action for all uploads with only one match'
```

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl [hunk 2]
```diff
@@ -148,11 +148,27 @@ resource configservice_label duplicate_asset_modal_next {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Gør for alle'
```
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Gentag denne handling for alle uploads med kun ét match'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl [hunk 3]
```diff
@@ -148,11 +148,27 @@ resource configservice_label duplicate_asset_modal_next {
...
+  ]
+}
+
+resource configservice_label duplicate_asset_modal_do_for_all {
+  key = 'DUPLICATE_ASSET_MODAL_DO_FOR_ALL'
+  group = 'Duplicate Assets'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Do for all'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Gentag denne handling for alle uploads med kun ét match'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl [hunk 4]
```diff
@@ -148,11 +148,27 @@ resource configservice_label duplicate_asset_modal_next {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Gør for alle'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Gentag denne handling for alle uploads med kun ét match'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 1 above)
Comment (Svane20) [reply MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]: I fear that the name is too long

Comment (Svane20) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]: Repeat this action for all uploads

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 3 above)
Comment (Svane20) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]: Repeat this action for all uploads

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 4 above)
Comment (Svane20) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]: Gentag denne handling for alle uploads

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 2 above)
Comment (Svane20) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]: Gentag denne handling for alle uploads

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 1 above)
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Repeat this action for all uploads'
```

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 2 above)
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Gentag denne handling for alle uploads'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 3 above)
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Repeat this action for all uploads'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 4 above)
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Repeat this action for all uploads'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 1 above)
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Repeat action for all'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 3 above)
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:167]:
```suggestion
      default_translation = 'Repeat action for all''
```

File: MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 2 above)
Comment (Poiar) [inline MM/5.10.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Gentag handling for alle'
```

File: MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl (hunk 4 above)
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Duplicate Assets.dcl:171]:
```suggestion
      default_translation = 'Gentag handling for alle'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**